proposals/vector_to_points.json

```
@@ -0,0 +1,68 @@
+{
+    "id": "vector_to_points",
+    "summary": "Sample random points from geometries",
+    "description": "For each geometry samples a number of random points based on one of the sampling method.",
```

Yeah, we could either append an "s" or use your suggestion

```suggestion
    "description": "For each geometry samples a number of random points based on a sampling method.",
```